server/etcdserver/api/etcdhttp/health.go

```diff
@@ -241,7 +268,7 @@ func (reg *CheckRegistry) InstallHttpEndpoints(lg *zap.Logger, mux *http.ServeMu
 }
 
 func (reg *CheckRegistry) runHealthChecks(ctx context.Context, checkNames ...string) Health {
-	h := Health{Health: "true"}
+	h := Health{Status: HealthStatusSuccess}
```

Comment:
Let's not to break the existing `/health` endpoint in 3.6.
```suggestion
	h := Health{Health: "true", Status: HealthStatusSuccess}
```

Reply:
this is not used in /health endpoint. It is only used for livez/healthz.